src/config.rs

```diff
@@ -978,3 +985,9 @@ impl From<Palette> for term::color::ColorPalette {
         p
     }
 }
+
+impl From<Palette> for term::color::ColorPalette {
+    fn from(cfg: Palette) -> term::color::ColorPalette {
+        (&cfg).into()
+    }
+}
```

Comment:
I'm kinda surprised this is not implemented automatically.

Reply:
The trick for this turned out to be:

```
  self.colors
            .as_ref()
            .cloned() // <--- !!
            .map(Into::into)
            .unwrap_or_else(Default::default)
```